<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rename</class>
 <widget class="QMainWindow" name="Rename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rename</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rename.qrc">
    <normaloff>:/rename.ico</normaloff>:/rename.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #333, stop: 0.01 #777, stop: 0.45 #999, stop: 0.99 #777, stop: 1 #333);
border-color: #333;
border-style: solid;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>1</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>23</y>
      <width>677</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>1</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>284</y>
      <width>696</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>23</y>
      <width>155</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(132, 132, 132);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>12</y>
       <width>35</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Files:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>66</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rename:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>93</y>
       <width>38</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Error:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>39</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Dirs:</string>
     </property>
    </widget>
    <widget class="QLabel" name="numfls">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>12</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numdir">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>39</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numrnm">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>66</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numerr">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>93</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>120</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip:</string>
     </property>
    </widget>
    <widget class="QLabel" name="numskp">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>185</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textEdit">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>310</y>
      <width>693</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 170, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rename.qrc"/>
 </resources>
 <connections/>
</ui>
